lib/sync/worker.js

```diff
@@ -80,4 +83,12 @@ QueueWorker.prototype.work = function() {
   });
 };
 
+/**
+ * Stop the queue worker
+ */
+QueueWorker.prototype.stop = function(cb){
```

Comment:
although the implementation is synchronous, I would like to give it a callback style interface, just incase this could change in the future.